server/instance_cloud.go

```diff
@@ -133,14 +134,17 @@ func (jci jiraCloudInstance) GetJIRAClient(jiraUser JIRAUser) (*jira.Client, err
 func (jci jiraCloudInstance) getJIRAClientForUser(jiraUser JIRAUser) (*jira.Client, *http.Client, error) {
 	oauth2Conf := oauth2_jira.Config{
 		BaseURL: jci.GetURL(),
-		Subject: jiraUser.Name,
+		Subject: jiraUser.AccountID,
```

Comment:
Apologies for the non-surgical change here, but I was going through all fields in the structs, and this just seemed much more intention-revealing.